test/integration/workarea/api/admin/payment_transactions_integration_test.decorator

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+module Workarea
+  module Api
+    module Admin
+      decorate PaymentTransactionsIntegrationTest, with: :kount do
+        def create_transaction
+          order = create_order
+          payment = create_payment(id: order.id)
```

Comment:
This is the change I made, to use `id: order.id` instead of the default which was apparently a BSON ID.